pkg/util/helper/binding.go

```diff
@@ -45,19 +48,12 @@ func (p ClusterWeightInfoList) Less(i, j int) bool {
 	if p[i].Weight != p[j].Weight {
 		return p[i].Weight > p[j].Weight
 	}
-	return p[i].ClusterName < p[j].ClusterName
-}
-
-// SortClusterByWeight sort clusters by the weight
-func SortClusterByWeight(m map[string]int64) ClusterWeightInfoList {
```

Comment:
This function is never used, just only used in one test case, so I moved it to UT code.